src/lib/components/ContextualLinks.svelte

```
@@ -0,0 +1,36 @@
+	{#if isOpen}
+		<div class="absolute z-10 mt-1 w-48 rounded border bg-white p-2 shadow-lg">
+			{#each searchLinks as link}
+				<a
+					href={link.url}
+					target="_blank"
+					rel="noopener noreferrer"
+					class="block p-1 hover:text-blue-500"
+				>
+					{link.name}
+				</a>
+			{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)